### ISSUE TITLE
Variable Importer issues with special characters

### ISSUE THREAD

**Issue by orson1282** [Silly-V/Adobe-Illustrator#49]
First of all, thanks a million for this script. It's amazing!! 👍 

I'm using Variable Importer version 8.2.4  with Illustrator CC 2015.
Everything works as expected except for csv files that contain names with special characters (é  ó)

For example:
Lyriénne becomes LyriÃ©nne
Falcón becomes FalcÃ³n

I noticed if I put these names at the top of the csv, it works fine,
If they're somewhere in between a csv file with 100 names it fails.
If they're at the end of a csv file with 100 names it also fails.

Any help would be highly appreciated.

**Comment by Silly-V** [comment]
Hello, would you be able to attach a sample CSV with the special characters so that I can try and reproduce the issue?

**Comment by Silly-V** [comment]
So, got the file?

**Comment by orson1282** [comment]
Yes I do, sorry for not getting back. I will send it tomorrow, need to sanitize the data.

**Comment by orson1282** [comment]
Here is the sample data and the ai file used.
[art_and_sample_data.zip](https://github.com/Silly-V/Adobe-Illustrator/files/6293570/art_and_sample_data.zip)

Thank you very much for the help. 👍

**Comment by Silly-V** [comment]
Okay I will take a little while, until I can get a good look but this is an essential start to begin doing some troubleshooting. In best case, script can be adjusted to accommodate your case, but at worst it could be an unmoveable native Illustrator error that needs to be reported to the Illustrator.uservoice website.
Thanks for providing the attachment!
